app/controllers/api/v1/messages_controller.rb

```diff
@@ -1,8 +1,7 @@
 class API::V1::MessagesController < ApplicationController
-    def index
-        @message = Message.all[Random.new.rand(1..5)]
+  def index
+    @message = Message.find(rand(1..5))
```

Comment:
- [ ] It will be great to use `Message.all.sample` instead of `Message.find(rand(1..5))` the reason is if you don't have messages with IDs from one to five, the application is failing. For example, I had only one message in my database and the application was checking for at least five messages, making the application, not function as implemented.